src/components/shared/MarkerTooltipContents.js

```diff
@@ -99,6 +99,14 @@ function getMarkerDetails(data: MarkerPayload): React.Element<any> | null {
                         nursery.lazy_capacity,
                         formatBytes
                       )}
+                  {nursery.chunk_alloc_us === undefined
+                    ? null
+                    : _markerDetail(
+                        'gctimeinchunkalloc',
+                        'Time spent allocating chunks in mutator',
+                        nursery.chunk_alloc_us,
+                        formatNumber
```

Comment:
should we give the unit too? Like replacing `nursery.chunk_alloc_us` with `formatNumber(nursery.chunk_alloc_us) + ' μs'` ?

Reply:
Yes, I should have done that :-(